hackathon/views.py

```diff
@@ -17,6 +17,81 @@ class HackathonListView(ListView):
 
 
 @login_required
+def judging(request, hack_id, team_id):
+    """Displays the judging page for the judge to save their scores
+    for the selected project - determined by hackathon id and team id"""
+
+    # HackProjectScoreCategories:
+    score_categories = HackProjectScoreCategory.objects.all()
+
+    event = Hackathon.objects.filter(pk=hack_id)
+    team = HackTeam.objects.filter(pk=team_id)
+
+    # verify whether user is judge for the event
+    user_is_judge = False
+    for judge in event.values('judges'):
+        if judge['judges'] == request.user.id:
+            user_is_judge = True
+    if not user_is_judge:
+        messages.error(request, "You are not a judge for that event!")
+        template = 'home/index.html'
+        return render(request, template)
+
+    # verify if event is ready to be judged (finished)
+    finish = event.values('end_date')[0]['end_date']
```

Comment:
Not sure this is the right trigger. If the event is finished, it is over and done with I would think. Maybe instead of that we can add another status to the hackathon called `judging` and add that.